services/storage/src/simcore_service_storage/modules/db/access_layer.py

```diff
@@ -261,7 +262,7 @@ async def get_project_access_rights(
         # determine user's access rights by aggregating AR of all groups
         return _aggregate_access_rights(row.access_rights, user_group_ids)
 
-    async def get_file_access_rights(
+    async def get_file_access_rights(  # pylint:disable=too-many-return-statements # noqa: PLR0911
```

Comment:
there is a very big if else here. maybe you could split that in 2 and make the intent clearer instead of disabling that warning. the function is pretty long